main.js

```diff
@@ -85,12 +68,69 @@ function makeNewActivity(event) {
   )
 }
 
+function validateInputs() {
+  validateDescription(); // had to do function for each otherwise it wouldn't display all of them
+  validateMinutes();
+  validateSeconds();
+  validateCategory();
+}
+
+function checkInputs() {
+  if (validateCategory() || validateDescription() || validateMinutes() || validateSeconds()) {
+    console.log('falseTest');
+    return false;
+  } console.log('trueTest');
+  return true;
+}
+function validateCategory() {
+ if (currentActivity.category === "") {
+   displayErrorMessage('category');
+   return true;
+ }
+}
+
+function validateDescription() {
+ if (currentActivity.description === "") {
+   displayErrorMessage('description');
+   changeInputColor('description');
+   return true;
+ }
+}
+
+function validateMinutes() {
+ if (currentActivity.minutes === "") {
+   displayErrorMessage('minutes');
+   changeInputColor('minutes');
+   return true;
+ }
+}
+
+function validateSeconds() {
+ if (currentActivity.seconds === "") {
+   displayErrorMessage('seconds');
+   changeInputColor('seconds');
+   return true;
+ }
+}
+
```

Comment:
Wow awesome job with these @jordy1611. Since they are all essentially the same shell function, we can just make a parameter called input, and pass either description, minutes or seconds in when we call the function. Like so:
`function validateInput(input) {
    if (currentActivity.input === "") {
        displayErrorMessage(`${input}`);
        changeInputColor(`${input}`);
        return true;
     }
 }`

Reply:
This is great Jordy! I really appreciate seeing the code broken down like this before we go in and refactor.